modules/scodec/shared/src/main/scala/io/circe/scodec/package.scala

```diff
@@ -33,3 +85,5 @@ package object scodec {
   implicit final val encodeByteVector: Encoder[ByteVector] =
     Encoder.encodeString.contramap(_.toBase64)
 }
+
+package object scodec extends ByteVectorCodec with BitVectorCodec
```

Comment:
I decided to split `Bit...` and `ByteVector` codecs since `BitVector` one got quite big. I find new structure more readable.